pkgs/development/libraries/libfsm/default.nix

```diff
@@ -22,11 +22,8 @@ stdenv.mkDerivation rec {
   # if we use stdenv vs clangStdenv, we don't know which, and CC=cc in all
   # cases.) it's unclear exactly what should be done if we want those flags,
   # but the defaults work fine.
-  buildPhase = "PREFIX=$out bmake -r -j$NIX_BUILD_CORES";
-  installPhase = ''
-    PREFIX=$out bmake -r install
-    runHook postInstall
-  '';
+  make = "bmake";
```

Comment:
I've never seen this before, but apparently there's some mention of `$make` elsewhere for use in FreeBSD's builders? Not that this necessarily should hold this back, but if you know the specifics it would be interesting.

Reply:
That's a custom `derivation {}`, so not really related to this at all.  It's just used to refer to the bootstrap `make` in stdenv AFAICT.

But you did make me realise that I was still setting `make` here, which is no longer necessary since we're now using a setup-hook.